psi4/driver/procrouting/proc.py

```diff
@@ -2514,16 +2514,13 @@ def run_cc_property(name, **kwargs):
     all CC property calculations.
 
     """
-    optstash = p4util.OptionsState(
-        ['WFN'],
-        ['DERTYPE'],
-        ['ONEPDM'],
-        ['PROPERTY'],
-        ['CCLAMBDA', 'R_CONVERGENCE'],
-        ['CCEOM', 'R_CONVERGENCE'],
-        ['CCEOM', 'E_CONVERGENCE'])
+    optstash = p4util.OptionsState(['WFN'], ['DERTYPE'], ['ONEPDM'], ['PROPERTY'], ['CCLAMBDA', 'R_CONVERGENCE'],
```

Comment:
For whatever reason, we usually don't format this as I think the other way seems cleaner.

Reply:
Should I just block it off with `yapf: enable/disable`?

Reply:
add `# yapf: disable` at the end of the line so it's self-contained.